contracts/pool-manager/sim/lp_fee.py

```
@@ -0,0 +1,178 @@
+        D_P = D
+        for _x in xp:
+            D_P = D_P * D / (_x * N_COINS)
+        Dprev = D
+
+        print("Dprev", Dprev)
+        D = (Ann * S + D_P * N_COINS) * D / ((Ann - 1) * D + (N_COINS + 1) * D_P)
+        print("D", D)
+        print("---")
+
+        # Equality with the precision of 1
+        if D > Dprev:
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `Dprev < D`

(SIM300)